<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Napatahti</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/app.ico</normaloff>:/app.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="vbox">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="sizeLabel">
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string notr="true">1000x390</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tableLabel">
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string notr="true">Current table</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSaveAsCalendar">
     <property name="title">
      <string>Moon Calendar</string>
     </property>
     <addaction name="actSaveAsCalendar"/>
    </widget>
    <widget class="QMenu" name="menuSaveAsCore">
     <property name="title">
      <string>Core statistic</string>
     </property>
     <addaction name="actSaveAsCoreStat"/>
    </widget>
    <widget class="QMenu" name="menuSaveAsLineBar">
     <property name="title">
      <string>Planet statistic</string>
     </property>
     <widget class="QMenu" name="menuSaveAsPrime">
      <property name="title">
       <string>Prime</string>
      </property>
      <addaction name="actSaveAsPrime"/>
     </widget>
     <widget class="QMenu" name="menuSaveAsAsha">
      <property name="title">
       <string>Asha</string>
      </property>
      <addaction name="actSaveAsAsha"/>
     </widget>
     <widget class="QMenu" name="menuSaveAsCosmic">
      <property name="title">
       <string>Cosmic</string>
      </property>
      <addaction name="actSaveAsCosmic"/>
     </widget>
     <addaction name="menuSaveAsPrime"/>
     <addaction name="menuSaveAsAsha"/>
     <addaction name="menuSaveAsCosmic"/>
    </widget>
    <addaction name="actNew"/>
    <addaction name="actEdit"/>
    <addaction name="menuSaveAsCalendar"/>
    <addaction name="menuSaveAsLineBar"/>
    <addaction name="menuSaveAsCore"/>
    <addaction name="actSaveAsMap"/>
    <addaction name="actPrintMap"/>
    <addaction name="separator"/>
    <addaction name="actAppQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="trigViewToolBar"/>
    <addaction name="separator"/>
    <addaction name="trigViewHeader"/>
    <addaction name="trigViewRecepTree"/>
    <addaction name="trigViewEsseStat"/>
    <addaction name="trigViewCircle"/>
    <addaction name="trigViewCrdTable"/>
    <addaction name="trigViewLineBar"/>
    <addaction name="trigViewAspCfg"/>
    <addaction name="trigViewAspStat"/>
    <addaction name="trigViewCoreStat"/>
    <addaction name="separator"/>
    <addaction name="trigViewStatusBar"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuMoonDay">
     <property name="title">
      <string>Moon Day</string>
     </property>
     <addaction name="trigModeJyotisa"/>
     <addaction name="trigModeWestern"/>
    </widget>
    <widget class="QMenu" name="menuCircle">
     <property name="title">
      <string>Zodiac circle</string>
     </property>
     <widget class="QMenu" name="menuZeroAng">
      <property name="title">
       <string>Zero angle</string>
      </property>
      <addaction name="trigZeroAngAsc"/>
      <addaction name="trigZeroAngAri"/>
     </widget>
     <widget class="QMenu" name="menuAccKey">
      <property name="title">
       <string>Accuracy</string>
      </property>
      <addaction name="trigAccMid"/>
      <addaction name="trigAccMax"/>
      <addaction name="trigAccMin"/>
      <addaction name="trigAccAny"/>
     </widget>
     <addaction name="menuAccKey"/>
     <addaction name="menuZeroAng"/>
     <addaction name="trigCircleDegrees"/>
     <addaction name="trigCircleSpeeds"/>
     <addaction name="trigCircleCuspids"/>
     <addaction name="trigCircleColors"/>
    </widget>
    <widget class="QMenu" name="menuCrdTable">
     <property name="title">
      <string>Coordinates</string>
     </property>
     <addaction name="trigFixedStars"/>
    </widget>
    <widget class="QMenu" name="menuCoreStat">
     <property name="title">
      <string>Core statistic</string>
     </property>
     <addaction name="trigModeGloba"/>
     <addaction name="trigModeSchit"/>
     <addaction name="separator"/>
     <addaction name="trigCoreStatCosEnb"/>
     <addaction name="trigCoreStatHorEnb"/>
     <addaction name="trigCoreStatInvert"/>
    </widget>
    <widget class="QMenu" name="menuLineBar">
     <property name="title">
      <string>Planet Statistic</string>
     </property>
     <widget class="QMenu" name="menuLineBarMode">
      <property name="title">
       <string>Method</string>
      </property>
      <addaction name="trigModePrime"/>
      <addaction name="trigModeAsha"/>
      <addaction name="trigModeCosmic"/>
     </widget>
     <widget class="QMenu" name="menuLineBarPrime">
      <property name="title">
       <string>Prime</string>
      </property>
      <addaction name="trigPrimeCosDet"/>
      <addaction name="trigPrimeHorDet"/>
      <addaction name="trigPrimeNormal"/>
     </widget>
     <widget class="QMenu" name="menuLineBarAsha">
      <property name="title">
       <string>Asha</string>
      </property>
      <addaction name="trigAshaPowKind"/>
      <addaction name="trigAshaPowCreat"/>
      <addaction name="trigAshaKindDet"/>
     </widget>
     <widget class="QMenu" name="menuLineBarCosmic">
      <property name="title">
       <string>Cosmic</string>
      </property>
      <addaction name="trigCosmicSum"/>
      <addaction name="trigCosmicDet"/>
     </widget>
     <addaction name="menuLineBarMode"/>
     <addaction name="menuLineBarPrime"/>
     <addaction name="menuLineBarAsha"/>
     <addaction name="menuLineBarCosmic"/>
    </widget>
    <widget class="QMenu" name="menuEsseStat">
     <property name="title">
      <string>Essentials</string>
     </property>
     <addaction name="trigBySunSpec"/>
     <addaction name="trigByEsseSpec"/>
     <addaction name="trigByAspSpec"/>
     <addaction name="trigDorAuriga"/>
     <addaction name="trigByAshaSpec"/>
    </widget>
    <widget class="QMenu" name="menuAspCfg">
     <property name="title">
      <string>Aspect configs</string>
     </property>
     <addaction name="trigAspCfgClassicOnly"/>
     <addaction name="trigAspCfgBonesOnly"/>
    </widget>
    <addaction name="menuMoonDay"/>
    <addaction name="menuEsseStat"/>
    <addaction name="menuCircle"/>
    <addaction name="menuCrdTable"/>
    <addaction name="menuLineBar"/>
    <addaction name="menuAspCfg"/>
    <addaction name="menuCoreStat"/>
    <addaction name="separator"/>
    <addaction name="actSettings"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actDataBase"/>
    <addaction name="actAspPage"/>
    <addaction name="actPlanets"/>
    <addaction name="actAspTable"/>
    <addaction name="actTCounter"/>
   </widget>
   <widget class="QMenu" name="menuConfigs">
    <property name="title">
     <string>Profile</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuConfigs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actEdit"/>
   <addaction name="actDataBase"/>
   <addaction name="actAspPage"/>
   <addaction name="actPlanets"/>
   <addaction name="actAspTable"/>
   <addaction name="actTCounter"/>
   <addaction name="actSettings"/>
   <addaction name="separator"/>
   <addaction name="actAppQuit"/>
  </widget>
  <action name="actAppQuit">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/exit.png</normaloff>:/24x24/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actSaveAsMap">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actNew">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/new.png</normaloff>:/24x24/new.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actEdit">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/edit.png</normaloff>:/24x24/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit...</string>
   </property>
   <property name="toolTip">
    <string>Edit...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actDataBase">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/dbase.png</normaloff>:/24x24/dbase.png</iconset>
   </property>
   <property name="text">
    <string>Data base...</string>
   </property>
   <property name="toolTip">
    <string>Data base...</string>
   </property>
  </action>
  <action name="actAspPage">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/report.png</normaloff>:/24x24/report.png</iconset>
   </property>
   <property name="text">
    <string>Aspect pages...</string>
   </property>
   <property name="toolTip">
    <string>Aspect pages...</string>
   </property>
  </action>
  <action name="actPlanets">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/list.png</normaloff>:/24x24/list.png</iconset>
   </property>
   <property name="text">
    <string>Planet catalog...</string>
   </property>
   <property name="toolTip">
    <string>Planet catalog...</string>
   </property>
  </action>
  <action name="actAspTable">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/table.png</normaloff>:/24x24/table.png</iconset>
   </property>
   <property name="text">
    <string>Aspect table...</string>
   </property>
   <property name="toolTip">
    <string>Aspect table...</string>
   </property>
  </action>
  <action name="actTCounter">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/clock.png</normaloff>:/24x24/clock.png</iconset>
   </property>
   <property name="text">
    <string>Time counter...</string>
   </property>
   <property name="toolTip">
    <string>Time counter...</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/setting.png</normaloff>:/24x24/setting.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="toolTip">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actPrintMap">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/print.png</normaloff>:/24x24/print.png</iconset>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="toolTip">
    <string>Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="trigViewToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="trigViewStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Statusbar</string>
   </property>
   <property name="toolTip">
    <string>Statusbar</string>
   </property>
  </action>
  <action name="trigViewHeader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Header</string>
   </property>
   <property name="toolTip">
    <string>Header</string>
   </property>
  </action>
  <action name="trigViewRecepTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reception tree</string>
   </property>
  </action>
  <action name="trigViewCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zodiac circle</string>
   </property>
  </action>
  <action name="trigViewEsseStat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Essentials</string>
   </property>
   <property name="toolTip">
    <string>Essentials</string>
   </property>
  </action>
  <action name="actSaveAsCalendar">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="trigModeJyotisa">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Jyotisa</string>
   </property>
  </action>
  <action name="trigModeWestern">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Western</string>
   </property>
  </action>
  <action name="trigCircleDegrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Degrees</string>
   </property>
  </action>
  <action name="trigCircleSpeeds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speeds</string>
   </property>
  </action>
  <action name="trigCircleCuspids">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cuspids</string>
   </property>
  </action>
  <action name="trigZeroAngAsc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ascendent</string>
   </property>
  </action>
  <action name="trigZeroAngAri">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0° Aries</string>
   </property>
   <property name="toolTip">
    <string>0° Aries</string>
   </property>
  </action>
  <action name="trigCircleColors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colors</string>
   </property>
   <property name="toolTip">
    <string>Colors</string>
   </property>
  </action>
  <action name="trigFixedStars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fixed stars</string>
   </property>
  </action>
  <action name="trigViewCrdTable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Coordinates</string>
   </property>
  </action>
  <action name="trigViewCoreStat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Core statistic</string>
   </property>
  </action>
  <action name="trigViewLineBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Planet statistic</string>
   </property>
   <property name="toolTip">
    <string>Planet statistic</string>
   </property>
  </action>
  <action name="trigViewAspCfg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aspect configs</string>
   </property>
  </action>
  <action name="trigViewAspStat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aspect statistic</string>
   </property>
  </action>
  <action name="trigModeGloba">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Globa PP</string>
   </property>
   <property name="toolTip">
    <string>Globa PP</string>
   </property>
  </action>
  <action name="trigModeSchit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Schitov BB</string>
   </property>
   <property name="toolTip">
    <string>Schitov BB</string>
   </property>
  </action>
  <action name="trigCoreStatCosEnb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cosmogram</string>
   </property>
   <property name="toolTip">
    <string>Cosmogram</string>
   </property>
  </action>
  <action name="trigCoreStatHorEnb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horoscope</string>
   </property>
   <property name="toolTip">
    <string>Horoscope</string>
   </property>
  </action>
  <action name="trigCoreStatInvert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invert</string>
   </property>
   <property name="toolTip">
    <string>Invert</string>
   </property>
  </action>
  <action name="actSaveAsCoreStat">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="trigModePrime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Prime</string>
   </property>
  </action>
  <action name="trigModeAsha">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Asha</string>
   </property>
  </action>
  <action name="trigModeCosmic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cosmic</string>
   </property>
  </action>
  <action name="trigPrimeCosDet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cosmogram</string>
   </property>
  </action>
  <action name="trigPrimeHorDet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horoscope</string>
   </property>
  </action>
  <action name="trigPrimeNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Both</string>
   </property>
  </action>
  <action name="actSaveAsPrime">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actSaveAsAsha">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actSaveAsCosmic">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="trigAshaPowKind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Power / Kindness</string>
   </property>
  </action>
  <action name="trigAshaKindDet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kindness detailed</string>
   </property>
  </action>
  <action name="trigAshaPowCreat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Power / Creative</string>
   </property>
  </action>
  <action name="trigBySunSpec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sun relation zones</string>
   </property>
  </action>
  <action name="trigByEsseSpec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cosmic special planets</string>
   </property>
  </action>
  <action name="trigDorAuriga">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Doriphoros / Auriga</string>
   </property>
   <property name="toolTip">
    <string>Doriphoros / Auriga</string>
   </property>
  </action>
  <action name="trigByAspSpec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aspect special planets</string>
   </property>
  </action>
  <action name="trigByAshaSpec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Asha special planets</string>
   </property>
  </action>
  <action name="trigAspCfgClassicOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Classic only</string>
   </property>
  </action>
  <action name="trigAspCfgBonesOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bones only</string>
   </property>
  </action>
  <action name="trigCosmicSum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </action>
  <action name="trigCosmicDet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detailed</string>
   </property>
  </action>
  <action name="trigAccMid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="trigAccMax">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Maximum</string>
   </property>
  </action>
  <action name="trigAccMin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimum</string>
   </property>
  </action>
  <action name="trigAccAny">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
  </action>
  <action name="actSaveAsProfile">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/save.png</normaloff>:/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actDeleteProfile">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/24x24/erase.png</normaloff>:/24x24/erase.png</iconset>
   </property>
   <property name="text">
    <string>Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
